packages/puppeteer-core/src/api/Page.ts

```diff
@@ -2873,6 +2874,14 @@ export class Page extends EventEmitter {
   waitForDevicePrompt(): Promise<DeviceRequestPrompt> {
     throw new Error('Not implemented');
   }
+
+  [Symbol.dispose](): void {
+    return void this.close().catch(debugError);
```

Comment:
Remove return on Sync dispose.

Reply:
Why are we returning a promise when it should be void then? Code currently does that at run time.